### PR TITLE
Add hawkular's provider to guides

### PR DESCRIPTION
**Pull request by karelhala**
As I promised @martinpovolny in https://github.com/ManageIQ/manageiq-ui-classic/pull/361 here is guide on how to run hawkinit with and without answers file.

@Jiri-Kremser, @pilhuhn, @theute what do you think?

### CODE REVIEW

**Comment by martinpovolny** [comment]
@himdel: please, review

**Comment by Jiri-Kremser** [comment]
looks good. Thanks for it. Perhaps it would worth adding that the h-services are started with a pre-created user with credentials `jdoe:password`. I will add that as well in the hawkinit (it can print that into console).

**Comment by mtho11** [comment]
The only concern is that this is not a supported Hawkular project. 
Should this be part of the Hawkular portfolio before it's in a guide? [I use it myself and think it's great]

**Comment by karelhala** [comment]
@Jiri-Kremser good point! Updated the guide.

@mtho11 That's true, we should wait what @theute and @pilhuhn think about this and we can add some kind of disclosure that this tool is not fully supported and it's just for easy development setup.

File: providers/hawkular.md [hunk 1]
```diff
@@ -73,6 +73,24 @@ There are two options how to run hawkinit:
 }
 
  ```
+
+### Accessing Hawkular
+After **Hawkinit** tool will download, install and run docker files, wait couple of minutes before all services are up
```
Comment (Jiri-Kremser) [inline providers/hawkular.md]: `s/docker files/docker images/`

**Comment by pilhuhn** [comment]
I think Hawkinit is cool - personally I like a docker(-compose) based setup much more, as I can easily dispose it afterwards and the chance of port-conflicts are smaller. 

And we can certainly move that tool either as project to github.com/hawkular or as sub-project inside of github.com/hawkular/hawkular-services

**Comment by Jiri-Kremser** [comment]
> I like a docker(-compose) based setup much more, as I can easily dispose it afterwards and the chance of port-conflicts are smaller.

same here, docker-compose is super easy compared to docker-link-hell or openshift. Once the openshift template is done, I can add the support for it into hawkinit. You know, at the beginning user can select either docker-compose or openshift (+version) as the container orchestration framework, then the other questions can theoretically remain the same.

**Comment by himdel** [comment]
Ok, seems like my previous problem of not being able to access localhost:8080 is fixed now ..
Verified following the instructions does give me a provider I can add to manageiq :+1: .

Although as this is guides, and we probably can't expect all the devs to know *much* about docker, maybe you could add instructions on a) how to stop it (I assume that's just `docker stop $(docker ps | grep -e 'hawk\|cass' | awk '{ print $1 }')`, and b) how to start it again after it's been stoppped (or after reboot) (no idea there).

**Comment by Jiri-Kremser** [comment]
> a) how to stop it 

(SIGINT) Ctrl+C should work: pressed 1 time it tries to shutdown everything grafefuly, pressing it twice it's less merciful

> b) how to start it again after it's been stoppped

The idea right now is that it's stateless and once you stop it, you loose everything (because it's very cheap to spin new containers). But if you want to threat your h-provider as a "pet", you can. If you run the hawkinit, it says something like:

```bash
Later, you can find your hawkular-services listening on http://localhost:8080
Running 'docker-compose up --force-recreate' in directory: /tmp/tmp-11573k3ujXFLACh9z
```

If you navigate to `/tmp/tmp-11573k3ujXFLACh9z`, you can run `docker-dompose up` to start it again. This is not a standard use-case though. (or any other docker-compose command, like `docker-compose logs -f myCassandra`, `docker-compose exec hawkular /bin/bash`, etc.).

Btw. all the Cassandra images has the volume mounted from the host system.

```bash
volumes:
- /tmp/opt/data:/opt/data:rw
```

So the data (hawkular inventory, alerts and metrics) should survive even though you remove the previous containers and start a new ones. If you want to also clean the previous data, do `rm -rf /tmp/opt/data/*`

**Comment by himdel** [comment]
@Jiri-Kremser aaah, thanks, makes sense :) I'm ok with treating them as cattle, as long as we know that's the preferred way :).

@karelhala would you add something like that ^^^ to the PR please? :)

**Comment by himdel** [comment]
@martinpovolny LGTM, if you have the green button...

(Unless you also want `setenforce` mentioned...)

**Comment by martinpovolny** [comment]
Should we link from somewhere to the new article?

**Comment by himdel** [comment]
Oh, good point, from README?

**Comment by martinpovolny** [comment]
probably

**Comment by himdel** [comment]
definitely :) https://github.com/ManageIQ/guides/pull/192

**Pull request merged**